tests/integration/cmd_debug_test.go

```diff
@@ -77,22 +77,6 @@ var _ = Describe("odo debug command tests", func() {
 			stopChannel <- true
 		})
 
-		It("should expect a ws connection when tried to connect on default debug port locally", func() {
```

Comment:
Why was this removed? This is still a valid  scenario

Reply:
Tests that have been removed without a comment have a devfile counterpart already present hence can be deleted

Reply:
ok makes sense